lms/djangoapps/course_blocks/api.py

```diff
@@ -82,6 +92,9 @@ def get_course_blocks(
             BlockStructureManager.get_collected.  Can be optionally
             provided if already available, for optimization.
 
+        grading - Ensure hidden blocks are taken into account for grading
+            purposes.
```

Comment:
Nit: `include_hidden_graded_items` would be a more explicit name for this parameter, and more similar to the existing `allow_start_dates_in_future` parameter.

Actually, is `allow_start_dates_in_future` solving a similar problem already??

Reply:
I'll have to investigate this.

Reply:
> Actually, is allow_start_dates_in_future solving a similar problem already??

@viadanna Did you get a chance to investigate this?

Reply:
I did investigate this further, @bradenmacdonald but I wasn't able to reproduce the initial issue in the latest master or juniper.2

We are planning to pause this while we get a better grasp of what changed.

Reply:
> I did investigate this further, @bradenmacdonald but I wasn't able to reproduce the initial issue in the latest master or juniper.2

@viadanna Is this PR still relevant?

Reply:
I don't think so, I couldn't replicate the issue on those branches and the original ticket has been on hold upstream for a while now. I'm closing this.